docker-compose.yml: add /tmp/geth-dag as a docker volume

meowsbits opened the pull request:
This avoids duplicated effort creating the DAG
for mining every time on start up.

Date: 2022-07-27 11:38:49-07:00
Signed-off-by: meows <b5c6@protonmail.com>

meowsbits committed:
docker-compose.yml: add /tmp/geth-dag as a docker volume

This avoids duplicated effort creating the DAG
for mining every time on start up.

Date: 2022-07-27 11:38:49-07:00
Signed-off-by: meows <b5c6@protonmail.com>

diega commented:
This is really helpful. Could we move it into an ignored directory within the repo? That's because `$TMPDIR` resolves to a different value in each OS and I don't know if it's possible to resolve that env variable within _compose_

meowsbits committed:
.gitignore,docker-compose.yml: move shared dag dir to project root

This avoids any potential issues wit
h the temporary directory difference
between OSes.

Date: 2022-07-27 12:03:08-07:00
Signed-off-by: meows <b5c6@protonmail.com>

meowsbits commented:
Done :point_up:
